### PR TITLE
[[ Bug 18303 ]] Fix stack id dialog when creating script only stack

### PR DESCRIPTION
**Pull request by montegoulding**

### CODE REVIEW

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 805adf6

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 805adf6d4cacdde3a3449109437ade5a8351c497

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -6556,20 +6556,44 @@ Tags: ide
...
-   return the result
+   if the result is empty then
+      return it for value
+   else
+      return the result for error
+   end if
 end revIDENewScriptOnlyStack
 
+/*
+Creates a new script only stack in memory with a given scriot.
```
Comment (livecodepanos) [inline Toolset/libraries/revidelibrary.8.livecodescript]: There is a typo here (`scriot`) :)

Comment (montegoulding) [reply Toolset/libraries/revidelibrary.8.livecodescript]: I always make my stacks with extra scriot... don't you?

Comment (livecodepanos) [reply Toolset/libraries/revidelibrary.8.livecodescript]: hehe :)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 07af5b6

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok 07af5b6254b0a67484401781b80880b66a8d58a7

**Comment by peter-b** [comment]
@montegoulding Since this is going into the stable branch, can it have a bug report + release note, please?  I'm having great difficulty working out what this patch even fixes...

**Comment by montegoulding** [comment]
OK, sure. If you create a script only stack you will see an unnecessary answer dialog with the stack id after it is created.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 0de84b9

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 0de84b9edce94baef3fe6dfe3f6afecf56a0aad3

**Pull request merged**